### ISSUE TITLE
[FR] - Transform SEO Image dimensions incorrect/outdated?

### ISSUE THREAD

**Issue by jayhlee** [nystudio107/craft-seomatic#746]
### Question
If Transform SEO Image is enabled with Crop mode, the resultant transformed image is 1200x630. Based on Google's Structured Data docs, it looks like they recommend providing multiple images with the aspect ratios of 16x9, 4x3, and 1x1 for many schema markups like `Article`, `Event`, and `Product` (and probably more).

I was wondering if there was a reason one image with the ratio of 1200x630 it outputted rather than Google's recommendation for specific structured data markups.

If the current Transform SEO Image output it outdated, it'd be great to have SEOMatic display the recommended image(s) with its respective recommended image dimensions based on the `Main Entity of Page` that is selected in the Content SEO.

**Comment by khalwat** [comment]
@jayhlee can you link to these recommendations, please?

**Comment by khalwat** [comment]
ref: https://developers.google.com/search/docs/data-types/article#non-amp-sd

So these guidelines have indeed changed:

- Only a marked-up image that directly belongs to the article should be specified.
- Images should be at least 1200 pixels wide.
- Every page must contain at least one image (whether or not you include markup). Google will pick the best image to display in Search results based on the aspect ratio and resolution.
- Image URLs must be crawlable and indexable.
- Images must represent the marked up content.
- The file format must be supported by Google Images.
- For best results, provide multiple high-resolution images (minimum of 800,000 pixels when multiplying width and height) with the following aspect ratios: 16x9, 4x3, and 1x1.

SEOmatic meets all of the requirements, except for the last "for best results" requirement. It's a little unclear to me how Google will actually be using these images, but I'd like to see these get implemented.

**Comment by jayhlee** [comment]
@khalwat Awesome. FYI - It looks like Google recommends the same image aspect ratios for almost all schema markups, but the `Movie` markup is different. 

> Images must have a high resolution and have a 6:9 aspect ratio. While Google can crop images that are close to a 6:9 aspect ratio, images largely deviating from this ratio aren't eligible for the feature.

[Source](https://developers.google.com/search/docs/data-types/movie#movie)

**Comment by wmdhosting** [comment]
is there a way to change this 1200x630
As much i have seen  that size of image are croped in google results.. 

Best  results what i have seen on other pages. mainly wordpress and similar. 
 are from images 1x1 ratio..  1000x1000 or 1200x1200

**Comment by khalwat** [comment]
@wmdhosting yes I think providing multiple images in various aspect ratios, and letting Google pick the one it wants to use would be ideal

**Comment by wmdhosting** [comment]
OK :) but can you point me how..   
as any image i put its transformed to 1200x630

**Comment by khalwat** [comment]
You can set the SEO image to whatever you want using the Twig API:

https://nystudio107.com/docs/seomatic/using.html#general-variables

```twig
{% do seomatic.meta.seoImage(myUrl) %}
```

You can transform an asset into whatever sizing you want using Craft's native transforms, and then pass in the URL as above.
